service/controller/resource/tccp/error.go

```diff
@@ -20,6 +21,45 @@ var executionFailedError = &microerror.Error{
 	Kind: "executionFailedError",
 }
 
+// event... is an error type for situations where we want to create an Kubernetes event in operatorkit.
+var eventCFCreateError = &microerror.Error{
```

Comment:
I introduced eventErrors, those should be used in case we want to emit events via operatorkit. For such events we need to make sure Kind and Desc is set. Kind also starts with upper-case letter to make sure it aligns with other event reasons.